Foundation/NSGeometry.swift

```diff
@@ -11,6 +11,7 @@
     import Darwin
 #elseif os(Linux)
     import Glibc
+    import CoreFoundation
```

Comment:
@phausler is there any other way to `#include <float.h>` on linux? `DBL_EPSILON` is visible on OS X, but not on ubuntu...